e2e/e2e-test.js

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node --experimental-modules
+
+import Web3 from "web3"
+import ProviderEngine from "web3-provider-engine"
+import WebsocketSubprovider from "web3-provider-engine/subproviders/websocket.js"
+
+import TBTC from "../tbtc.js/index.js"
```

Comment:
Can we import it from a package dependency if we provide it in `package.json` as a path to tbtc.js?

```suggestion
import TBTC from "@keep-network/tbtc.js";
```

Reply:
Changed.